extension/app/ts/components/subcomponents/address.tsx

```diff
@@ -111,6 +87,7 @@ export type ActiveAddressParams = {
 	subtitle?: string
 	simulationMode: boolean
 	changeActiveAddress: () => void
+	renameAddressCallBack: RenameAddressCallBack
```

Comment:
This one isn't optional like many others.  What is the reason for that?  Also, I don't understand why this needs to be optional at all, what is the situation where you wouldn't have this set?

Reply:
It's not set in other than the extension popup, as I don't have system to show the edit address dialog elsewhere at the moment. I could have added it to this PR, but then the PR would be way bigger. Once its implemented everywhere, that doesn't need to be optional anymore